models/classes/user/CommentMentionUserSearchService.php

```
@@ -0,0 +1,318 @@
+
+        return [
+            'id' => $userResource->getUri(),
+            'login' => $login,
+            'displayName' => $this->resolveDisplayName($userResource, $login),
+        ];
+    }
+
+    private function resolveLogin(core_kernel_classes_Resource $userResource): ?string
+    {
+        $loginProperty = $this->ontology->getProperty(GenerisRdf::PROPERTY_USER_LOGIN);
+        $login = trim((string) $userResource->getOnePropertyValue($loginProperty));
```

PHPStan (2.2.8)

[error] Cannot cast core_kernel_classes_Container to string.


(cast.string)